cmd/influx/global.go

```diff
@@ -242,34 +246,94 @@ func commonFlags() []cli.Flag {
 	return append(commonFlagsNoToken(), commonTokenFlag())
 }
 
-// getOrgBucketFlags returns flags used by commands that are scoped to a single org/bucket, binding
+// getOrgFlags returns flags used by commands that are scoped to a single org, binding
 // the flags to the given params container.
-func getOrgBucketFlags(c *cmd.OrgBucketParams) []cli.Flag {
+func getOrgFlags(params *cmd.OrgParams) []cli.Flag {
 	return []cli.Flag{
-		&cli.GenericFlag{
-			Name:    "bucket-id",
-			Usage:   "The bucket ID, required if name isn't provided",
-			Aliases: []string{"i"},
-			Value:   &c.BucketID,
-		},
-		&cli.StringFlag{
-			Name:        "bucket",
-			Usage:       "The bucket name, org or org-id will be required by choosing this",
-			Aliases:     []string{"n"},
-			Destination: &c.BucketName,
-		},
 		&cli.GenericFlag{
 			Name:    "org-id",
 			Usage:   "The ID of the organization",
 			EnvVars: []string{"INFLUX_ORG_ID"},
-			Value:   &c.OrgID,
+			Value:   &params.OrgID,
 		},
 		&cli.StringFlag{
 			Name:        "org",
 			Usage:       "The name of the organization",
 			Aliases:     []string{"o"},
 			EnvVars:     []string{"INFLUX_ORG"},
-			Destination: &c.OrgName,
+			Destination: &params.OrgName,
+		},
+	}
+}
+
+// getBucketFlags returns flags used by commands that are scoped to a single bucket, binding
+// the flags to the given params container.
+func getBucketFlags(params *cmd.BucketParams) []cli.Flag {
+	return []cli.Flag{
+		&cli.GenericFlag{
+			Name:    "bucket-id",
+			Usage:   "The bucket ID, required if name isn't provided",
+			Aliases: []string{"i"},
+			Value:   &params.BucketID,
 		},
+		&cli.StringFlag{
+			Name:        "bucket",
+			Usage:       "The bucket name, org or org-id will be required by choosing this",
+			Aliases:     []string{"n"},
+			Destination: &params.BucketName,
+		},
+	}
+}
+
+// getOrgBucketFlags returns flags used by commands that are scoped to a single org/bucket, binding
+// the flags to the given params container.
+func getOrgBucketFlags(c *cmd.OrgBucketParams) []cli.Flag {
+	return append(getBucketFlags(&c.BucketParams), getOrgFlags(&c.OrgParams)...)
+}
+
+
+// readQuery reads a Flux query into memory from a file, args, or stdin based on CLI parameters.
+func readQuery(ctx *cli.Context) (string, error) {
+	nargs := ctx.NArg()
+	file := ctx.String("file")
```

Comment:
Feels like it'd be "cleaner" to have the flag name be a function arg, but if all the call-sites use the same name then maybe it's overkill? WDYT?

Reply:
I initially thought this, but then realised the error messages are all specific to flux queries too, so we should probably keep the args consistent for all flux queries in the cli